File: Assignment39_2.java
///////////////////////////////////////////////////////////////////////////////
// Function Name : Display
// Description   : Display numbers from 1 to 5 using recursion
// Input         : ---
// Output        : 1  2  3  4  5
// Author        : Raj Samir Jadhav
// Date          : 03/12/2025
///////////////////////////////////////////////////////////////////////////////
class Pattern
{
    static int i = 1;

    void Display()
    {
        if(i <= 5)
        {
            System.out.print(i + "  ");
            i++;
            Display();        // Recursive call
        }
    }
}

class Assignment39_2
{
    public static void main(String[] args)
    {
        Pattern obj = new Pattern();
        obj.Display();
    }
}

File: Assignment39_2.c
#include <stdio.h>

///////////////////////////////////////////////////////////////////////////////
// Function Name : Display
// Description   : Display numbers from 1 to 5 using recursion
// Input         : ---
// Output        : 1  2  3  4  5
///////////////////////////////////////////////////////////////////////////////
void Display()
{
    static int i = 1;

    if(i <= 5)         // Base condition
    {
        printf("%d  ", i);
        i++;
        Display();     // Recursive call
    }
}

int main()
{
    Display();
    return 0;
}
